linux_os/guide/system/auditing/package_audit-audispd-plugins_installed/rule.yml

```diff
@@ -1,5 +1,7 @@
 documentation_complete: true
 
+prodtype: fedora,ol9,rhel9,sle12,sle15,ubuntu1604,ubuntu1804,ubuntu2004
```

Comment:
we currently have this rule only in ubuntu2004

Reply:
Thanks. I will remove the previous versions.

Reply:
We will need to figure out something for RHEL 9 as the package doesn't exist there.

Reply:
Indeed, the package doesn't exist there. But the rule was already already in the data stream by the absence of `prodtype`.

Reply:
It is worth noting that the rule is not selected in any of our shipped RHEL 9 profiles as of b4612471aae69dd19eefed987038f1dbcd758237 . How about putting a comment to the rule.yaml file explaining why the rhel9 profile got there?

Reply:
Managed by b95070ca34d3fb44a8f91faf8f3aae2257485954 , 2f2e088ec6d9088fb532a88c13ef71b9b40cd8b8 and 2d4af5f493832a2df29fb633db11bed5a428a87c